apps/web/next.config.js

```diff
@@ -1,8 +1,18 @@
 /** @type {import('next').NextConfig} */
+const path = require('path');
 const webpack = require('webpack');
 
 module.exports = {
   webpack: (config) => {
+
+      // Add path aliases
+    config.resolve.alias = {
```

Comment:
does this not break existing codebase? If not going to mege it

Reply:
You can use aliases in new files or new parts of your codebase while keeping relative paths in existing files. This allows you to gradually transition to using aliases.

Reply:
You can have a mix of imports in your codebase. For example, some files might use import Button from '@components/Button' while others might use import Button from '../../components/Button'. This approach is perfectly valid.

Reply:
you can think of aliases in configuration as a form of syntactic sugar. They provide a more convenient and readable way to handle imports, especially in larger projects.